final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,150 @@
+import feedparser
+import argparse
+import html2text
+from requests import get
+from datetime import datetime
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--url', type=str, help='Please enter a valid URL for RSS Feed')
+parser.add_argument('--limit', type=int, default=1, help='You can set a limit for news')
+parser.add_argument('--verbose', type=bool, default=False, help='If you want to know '
+                                                                'what\'s happening, set this to True')
+parser.add_argument('--version', type=str, default=False, help='Print version of program')
+args = parser.parse_args()
+url = args.url
+verbose = args.verbose
+version = '[Reader version: 0.01]'
+
+
+def to_text(html, rehtml=False):
+    """
+    This function creates instance of html2text.HTML2text
+    and configuring it.
+    We will use this to cleanup all HTML markup in our feed
+    :param html: Text to format
+    :param rehtml: True or False
+    :return: formatted text
+    """
+    formatter = html2text.HTML2Text()
+    formatter.wrap_links = False
+    formatter.skip_internal_links = True
+    formatter.inline_links = True
+    formatter.ignore_anchors = True
+    formatter.ignore_images = True
+    formatter.ignore_emphasis = True
+    formatter.ignore_links = True
+    text = formatter.handle(html)
+    text = text.strip(' \t\n\r')
+    if rehtml:
+        text = text.replace('\n', '<br/>')
+        text = text.replace('\\', '')
+    return text
+
+
+def check_url(url: str, verbose: bool):
+    """
+    This function checks url provided by user
+    Function will check if Content-Type of document on url
+    is xml
+    :param url: url provided by user
+    :param verbose: True or False
+    :return: True in case of xml,
+    False and closing program in all other cases
+    """
+    if verbose:
+        print('Wait a second, we will check if there is any RSS in URL you have given')
+        print('-' * 50)
+    response = get(url)
+    r = response.headers['Content-Type']  # We will check only headers
+    if 'xml' not in r:
+        print('Something went wrong. \n'
+              'Looks like URL you\'ve entered doesn\'t content any RSS feed')
+        return False
+    else:
+        return True
+
+
+def get_feed(url: str, verbose: bool):
+    """
+    Caching feed by creating a feedparser instance.
+    Returns non-formatted feed
+    :param url: url provided by user
+    :param verbose: True of False
+    :return: non-formatted feed
+    """
+    if verbose:
```

Comment:
Here and below. For such logging purposes it is bettert to use `logging` standart library. Check it put